visualizer/src/css/styles.css

```
@@ -152,8 +152,9 @@ a {
 	max-width: 30vw;
 	max-height: 50vh;
 
-	background: color-mix(in srgb, var(--panel-bg) 77%, transparent);
-	backdrop-filter: blur(1px);
+	background: color-mix(in srgb, var(--panel-bg) 70%, transparent);
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)